### PR TITLE
feat: add BigQuery Studio embedded notebook showing analysis with Gemini models

### PR DESCRIPTION
**Pull request by aliciawilliams**
# This PR adds a notebook for using Gemini Pro and Gemini Pro Vision models in BQ using integration in BQML. Notebook is focused on BigQuery Studio embedded notebook experience.

Thank you for opening a Pull Request!
Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

- [X] Follow the [`CONTRIBUTING` Guide](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/CONTRIBUTING.md).
- [X] You are listed as the author in your notebook or README file.
  - [X] Your account is listed in [`CODEOWNERS`](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/.github/CODEOWNERS) for the file(s).
- [X] Make your Pull Request title in the <https://www.conventionalcommits.org/> specification.
- [X] Ensure the tests and linter pass (Run `nox -s format` from the repository root to format).
- [N/A] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/README.md
Comment (holtskinner) [inline gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/README.md:1]: I don't think the README is needed if the folder just contains a notebook

Comment (aliciawilliams) [reply gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/README.md:1]: That's fine with me. I just realized my name as author wasn't in the notebook, and didn't want to potentially cause a need to relint.

File: gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb [hunk 1]
```diff
@@ -0,0 +1,708 @@
+{
```
Comment (holtskinner) [inline gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb:1]: <p>Can you add a link to the video? (If it exists now)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (holtskinner) [inline gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb:1]: <p>Can you add the buttons used in other notebooks for this repo?</p><p> </p><p>Similar to <a href="https://github.com/GoogleCloudPlatform/generative-ai/blob/main/gemini/getting-started/intro_gemini_python.ipynb" target="_blank">https://github.com/GoogleCloudPlatform/generative-ai/blob/main/gemini/getting-started/intro_gemini_python.ipynb</a> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (holtskinner) [inline gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb:1]: <p>Can you make the gs:// link in code font?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (holtskinner) [inline gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb:1]: <p>Can you put the dataset name in code font?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (holtskinner) [inline gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb:1]: <p>I don't think the prompt is needed in two places, unless you can't read the code in BQ studio notebooks or something</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

File: gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/README.md
Comment (holtskinner) [reply gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/README.md:1]: I'd recommend adding your name to the notebook, similar to how other notebooks in this repo have it set up. https://github.com/GoogleCloudPlatform/generative-ai/blob/main/gemini/getting-started/intro_gemini_python.ipynb

**Review by holtskinner — APPROVED** [review]
Couple of minor nits. Looks good overall! Not sure if owlbot will do this automatically, but it would also be nice if you could format the SQL cells how bigquery auto-formats them.

File: gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb (hunk 1 above)
Comment (aliciawilliams) [reply gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb:1]: <p>There's isn't yet a "Run in BigQuery Studio" button though it is in the works. I really only want people to open in BigQuery Studio, so ideally that would be the only button. I can add an open in Colab Enterprise button if you want?</p>

Comment (aliciawilliams) [reply gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb:1]: <p>Video isn't ready, and so I will remove the line for now. Thanks for catching that!</p>

Comment (aliciawilliams) [reply gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb:1]: <p>I don't know a way to combine a link with code format in markdown, do you?</p>

Comment (aliciawilliams) [reply gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb:1]: <p>done!</p>

Comment (aliciawilliams) [reply gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb:1]: <p>Yeah, you can't read it in BQ Studio (I added this for my demo). I'll remove it in this version of the notebook.</p>

Comment (aliciawilliams) [reply gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb:1]: <p>I just added a "Run in Colab Enterprise" button, but plan to keep this paragraph. That work?</p>

Comment (holtskinner) [reply gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb:1]: Thats good

Comment (holtskinner) [reply gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb:1]: You can do this:

```
[`gs://link`](LiveURL)
```

[`gs://link`](LiveURL)

Comment (aliciawilliams) [reply gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb:1]: <p>Awesome - thank you!</p>

**Comment by aliciawilliams** [comment]
> Couple of minor nits. Looks good overall! Not sure if owlbot will do this automatically, but it would also be nice if you could format the SQL cells how bigquery auto-formats them.

I made updates and also formatted the queries - some don't format nicely in BQ editor but I used formats from the docs in those cases.

**Pull request merged**